apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -2967,6 +2967,11 @@ export const extensions: Extensions = {
                 replicateContent: {
                     header: "Replicate content?",
                     message: "Seems like you don't have any content for this locale. Do you need to populate the previous locale's content here as a quick start?"
+                },
+                updateFromRootOrg: {
```

Comment:
Any particular reason we have used `rootorg` phrase in i18n key? based on the context, I think the issue being addressed has nothing to do with organization type, has it?

Reply:
The issue has context related to root org. The email template for different locale can only be updated from the root org and only viewable from the sub org.